### ISSUE TITLE
kwargs['position_ids'] = position_embeds?

### ISSUE THREAD

**Issue by cq70605** [yhgon/LongRoPE#1]
` def forward(self, input_ids, attention_mask=None, position_ids=None, use_long_rope=False, **kwargs):
        if use_long_rope and position_ids is not None:
            position_embeds = long_rope(input_ids, position_ids, self.dim, base=self.base, lambda_factors=self.lambda_factors, n_prime=self.n_prime)
            kwargs['position_ids'] = position_embeds`

'position_ids' = position_embeds, is that correct?
